packages/react-server-cli/package.json

```diff
@@ -43,7 +43,7 @@
     "pem": "^1.8.1",
     "q": "1.4.1",
     "raw-loader": "^0.5.1",
-    "react-hot-loader": "~1.3.0",
+    "react-hot-loader": "^3.1.1",
```

Comment:
Is this version compatible with the older versions of React that are supported below?

Reply:
I added a comment below. Yes it is compatible, but we will lose client-side HMR because react-hot-loader now needs us to use `module.hot.accept` and rendering the component tree in our entrypoints rather than using the no longer supported `ReactMount`.
See step 2 here http://gaearon.github.io/react-hot-loader/getstarted/